packages/core/useAsyncState/demo.vue

```diff
@@ -1,7 +1,19 @@
 <script setup lang="ts">
-import { useAsyncState } from '@vueuse/core'
+import { reactify, useAsyncState } from '@vueuse/core'
 import axios from 'axios'
-import yaml from 'yaml'
+import YAML from 'yaml'
+
+const stringify = reactify(
```

Comment:
in this one we can probably just use `YAML.stringify(state)` since it doesn't contain any functions